### PR TITLE
Adding Hospital registration form to Project collection

### LINKED ISSUE #663: Add Hospital registration form validation in the project collection
## Introduction
Hey! I want to add a form registration through javascript in project collection as a participant of GSSoC. Please assign it to me!
Thanks and regards.

## Screenshot:
bandicam 2021-03-25 19-58-02-732

### PR DESCRIPTION
**Pull request by AyushPradhan9**
## Related Issue
Fixes #663 

Closes: #663 

### Describe the changes you've made
I have added a new project which is a registration form validation using javascript.

### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots
bandicam 2021-04-01 12-57-31-243
bandicam 2021-04-01 12-57-34-025

@urvashi-code1255 @Vishal-raj-1 @991rajat Please review the PR.

### CODE REVIEW

**Comment by 991rajat** [comment]
@AyushPradhan9 Resolve the conflict

**Comment by AyushPradhan9** [comment]
I have done some changes @991rajat please review it again. Thanks!

File: public/registration_form.html [hunk 1]
```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <title>Hospital Registration</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="../assets/css/registration_form.css">
+    <script type="text/javascript" src="../assets/js/registration_form.js"></script>
```
Comment (991rajat) [inline public/registration_form.html:8]: Always add script at the end of the body

**Comment by 991rajat** [comment]
@AyushPradhan9 After Submitting it redirect to somehere fix that,
Add something after submitting.

**Comment by AyushPradhan9** [comment]
Hey @991rajat I have made the changes. Please review the PR. Thanks!

File: public/registration_form.html (hunk 1 above)
Comment (AyushPradhan9) [reply public/registration_form.html]: Done the changes @991rajat .

**Comment by 991rajat** [comment]
Congratulations on merging of first PR. :tada: 
Keep Contributing.

**Pull request merged**